CMakeLists.txt

```diff
@@ -147,15 +147,16 @@ if (WHISPER_BLAS)
             message(WARNING "BLAS library was not found. Environment variable OPENBLAS_PATH not defined.")
         endif ()
     else ()
-        set(BLA_STATIC 1)
-        set(BLA_VENDOR ${WHISPER_BLAS_VENDOR})
+        set(BLAS_STATIC 1)
+        set(BLAS_VENDOR ${WHISPER_BLAS_VENDOR})
```

Comment:
Does not look right

https://cmake.org/cmake/help/latest/module/FindBLAS.html

Reply:
Thanks for catching that, reverted the offending lines.